content/BindingMethods/BindingMethods.q.md

```diff
@@ -47,23 +47,23 @@
     $data = $pstmt->fetchAll();
 
     foreach ($data as $row) {
-        echo $row['student_id'].$row['first_name'].$row['last_name'].$row['birth_date'].$row['gender']. ' ';
+        echo $row['student_id']."\t".$row['first_name']." ".$row['last_name']."\t".$row['birth_date']."\t".$row['gender']."\n";
     }
 ?>
 ```
-/// type=SS, answer=[5] 
+/// type=SS, answer=[3] 
 
-Execute the program. What is its output?
+Execute the program. What is printed on lines 2, 3, and 4?
 
-- It produces an error.
+- It prints `Unable to establish a connection.`.
 
-- No output is displayed.
+- It prints `Successfully connected to the database.`.
```

Comment:
I think the phrase `It prints` is redundant in the options here since we already asked `What is printed`. How about having the following:

What are printed on lines 2, 3, and 4?
 - `Unable to establish a connection.`

- `Successfully connected to the database.`

- `1 John Smith 1999-02-10 M`

- `2 Samantha Danes 1999-10-12 F`

- `6 Alisa Ells 1999-06-30 F`

Then, we will have an MS answers. Please let me know your thoughts on this. Thanks.